galaxy_importer/collection.py

```diff
@@ -47,22 +48,29 @@ def import_collection(filepath, logger=None):
 
 
 def _import_collection(filepath, logger):
-    filename = os.path.basename(filepath)
-
     with tempfile.TemporaryDirectory() as extract_dir:
         with tarfile.open(filepath, 'r') as pkg_tar:
             pkg_tar.extractall(extract_dir)
 
-        return CollectionLoader(extract_dir, filename, logger=logger).load()
+        data = CollectionLoader(extract_dir, filepath, logger=logger).load()
+
+        _run_post_load_plugins(
+            artifact_path=filepath,
```

Comment:
Why path to archive is passed to plugin, despite it's already extracted?

Reply:
@cutwater `ansible-test` requires the collection be in a specific working directory `{...}/ansible_collections/{namespace}/{collection}/` - passing the filepath allows the plugin to extract the artifact a second time to accommodate this

I see that `_run_post_load_plugins()` should be located out of the `extract_dir` context manager, I will update that.

Reply:
I wonder if `ansible-test` can simply create a symbolic link in a specific format to this directory?
Also this is probably a bug that should be reported to `ansible-test`, don't you think?

Reply:
@cutwater I'll see if the symlink will work.

The working directory is a current requirement of ansible-test:
```
(vans-e) awc:collection_reqs_test$ /home/awc/code/ansible/bin/ansible-test sanity --help
ERROR: The current working directory must be at or below:

 - the Ansible source: /home/awc/code/ansible/
 - an Ansible collection: {...}/ansible_collections/{namespace}/{collection}/

Current working directory: /home/awc/code/collection_reqs_test
```

Reply:
symlinks will not work with ansible-test since some of the underlying tests require being in the actual directory